civicase.civix.php

```diff
@@ -443,3 +443,22 @@ function _civicase_civix_civicrm_alterSettingsFolders(&$metaDataFolders = NULL)
     $metaDataFolders[] = $settingsDir;
   }
 }
+
+/**
+ * (Delegated) Implements hook_civicrm_entityTypes().
+ *
+ * Find any *.entityType.php files, merge their content, and return.
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_entityTypes
+ */
+
+function _civicase_civix_civicrm_entityTypes(&$entityTypes) {
```

Comment:
Where is this function used? And why do we need it?
This looks completely unrelated.

Reply:
Function was auto generated. I have removed it.